src/api/users/index.js

```diff
@@ -1,9 +1,9 @@
 const { Router } = require('express');
 const controller = require('./controller');
-
+const { rateLimiting } = require('../../utils/Middlewares');
 const router = Router();
 
 router.get('/stats/:githubId', controller.getUserStats);
-router.post('/sync-prs', controller.syncPullRequests);
+router.post('/sync-prs', verifyJWT , rateLimiting  ,controller.syncPullRequests);
```

Comment:
forgot to import `verifyJWT`

Reply:
Sorry Brother for this inconvenience i have updated this.